inspirehep/modules/theme/ext.py

```diff
@@ -59,6 +59,7 @@ def init_app(self, app, assets=None, **kwargs):
 
         # Add Jinja2 extensions.
         app.jinja_env.add_extension('jinja2.ext.do')
+        app.jinja_env.autoescape = False
```

Comment:
Actually we should really keep autoescape to `True` in order to avoid XSS slipping through